File: 2265-partition-array-according-to-given-pivot/2265-partition-array-according-to-given-pivot.java
class Solution {
    public int[] pivotArray(int[] nums, int pivot) {
      List<Integer> min=new ArrayList<>();
      List<Integer> max=new ArrayList<>();
      List<Integer> piv=new ArrayList<>();

      for(int x: nums){
        if(x<pivot) min.add(x);
        else if(x>pivot) max.add(x);
        else piv.add(x);
      }
      min.addAll(piv);
      min.addAll(max);

      int[] result = new int[min.size()];
      for (int i = 0; i < min.size(); i++) result[i] = min.get(i);

      return result;

    }
}

File: 2265-partition-array-according-to-given-pivot/2265-partition-array-according-to-given-pivot.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        less=[]
        more=[]
        eq=[]

        for x in nums:
            if x<pivot:
                less.append(x)
            elif x>pivot:
                more.append(x)
            else:
                eq.append(x)
            
        less.extend(eq)
        less.extend(more)

        return less
        